chrome/shared/util.ts

```diff
@@ -0,0 +1,15 @@
+export function unique<T>(array: T[]): T[] {
+  return array.filter((value, i) => array.indexOf(value) === i);
+}
+
+export function range(min: number, max: number): number[] {
+  return Array.from({ length: max - min + 1 }, (_, i) => i + min);
+}
+
+export function sample<T>(array: T[]): T {
+  return array[Math.floor(Math.random() * array.length)];
+}
+
+export function generateRandomNumber(min: number, max: number): number {
+  return sample(range(min, max));
```

Comment:
That is a beautiful bit of overengineering :smile:
:man_shrugging: The usual way of doing this is:
```suggestion
  return min + Math.floor(Math.random() * (max - min + 1));
```